templates/AureliaSpa/ClientApp/app/components/fetchdata/fetchdata.ts

```diff
@@ -0,0 +1,27 @@
+/// <reference path="../../../../node_modules/aurelia-fetch-client/doc/whatwg-fetch.d.ts" />
+/// <reference path="../../../../node_modules/aurelia-fetch-client/doc/url.d.ts" />
```

Comment:
These don't have anything to do with Aurelia as far as I know. They are standards that TypeScript is missing for some reason. However, I don't think that `reference` comments should be needed either way.